content/en/Platform Deep Dive/Pentests/Pentest Process/Methodologies/mobile.md

```diff
@@ -32,19 +32,106 @@ Learn more about these objectives from OWASP:
 - [OWASP Mobile Top 10](https://owasp.org/www-project-mobile-top-10)
 - [OWASP Application Security Verification Standard (ASVS)](https://owasp.org/www-project-application-security-verification-standard)
 
-We look at application logic by working with your app.
+### Share Mobile App Files
+
+We look at application logic by working with your app. To support our pentesters, share the .ipa (iOS) and/or the .apk (Android) 
+files when you [Define Your Assets](/getting-started/assets/). You may also share this information in the private Slack channel
+for your pentest.
+
+### Methodology Details
 
 We follow an industry standard methodology primarily based on the OWASP Application Security
 Verification Standard (ASVS) and Testing Guide. Our team takes the following steps to ensure
 full coverage:
 
-- Reconnaissance
-  - Share the mobile application files
-    - Android: .apk
-    - iOS: .ipa
-- Automated and Manual Testing
-- Exploit Discovered Vulnerabilities
-- Report, triage, and retest
+Select the <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> key for more information on each step.
```

Comment:
@mjang-cobalt I think there should be a way to use our #primary color. I experimented with HTML but didn't get the result we need.

Reply:
I _think_ the problem might be related to my use of Font Awesome icons (`fas`)